dev-resources/puppetlabs/puppetserver/shell_utils_test/list_env_var_names

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env ruby
+
+puts ENV.keys
```

Comment:
@camlow325 said we already have that dependency somewhere else, maybe for cert autosigning tests or something